src/txn_builder.rs

```diff
@@ -9,46 +9,27 @@
 //!
 //! This module defines the subset of ledger state required by a keystore to build transactions, and
 //! provides an interface for building them.
-use crate::{
-    events::EventIndex,
-    records::{Record, Records},
-    sparse_merkle_tree::SparseMerkleTree,
-    transactions::{SignedMemos, TransactionParams},
-};
+use crate::records::{Record, Records};
```

Comment:
What's still in this file? Any reason not to move the last few things to `ledger_state.rs` and get rid of this?